orc8r/cloud/configs/orchestrator.yml

```diff
@@ -14,8 +14,10 @@ useGRPCExporter: true
 prometheusGRPCPushAddress: "prometheus-cache:9092"
 
 analytics:
+  # Metrics in this Orchestrator configuration should strictly be generic in nature
```

Comment:
Similar to elsewhere -- if you're gonna wrap the comment (which you should since it's long), wrap at 79 chars for uniformity